web/src/components/molecules/Member/MemberTable/index.tsx

```
@@ -62,15 +62,37 @@ const MemberTable: React.FC<Props> = ({
+        title:
+          users.length > 1
+            ? t("Are you sure to remove these members?")
+            : t("Are you sure to remove this member?"),
         icon: <Icon icon="exclamationCircle" />,
-        content: t(
-          "Remove this member from workspace means this member will not view any content of this workspace.",
+        content: (
+          <>
+            <RemoveUsers>
+              {users.map(user => (
+                <RemoveUser>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)